### ISSUE TITLE
Accessibility: Talkback unable to move focus in the conversation screen

### ISSUE THREAD

**Issue by RastislavKish** [signalapp/Signal-Android#15027]
### Guidelines

- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

### Bug description

In the approx. last week or two, when I open any conversation in Signal, I'm no longer able to use the swipe left / right gestures of Talkback, which usually move the focus from element to element. The screenreader can apparently see the content, because when I drag my finger on the screen, it correctly reads the messages and other components I slide across, just the swipe left / right navigation (the usual way of operating the smartphone for a blind person) does not work for some reason.

## Reproduction steps:

1. Open Signal
2. Turn on Talkback
3. Use swipe right / left gestures to focus a conversation and double tap to activate it
4. Try to use swipe left / right again to move on the screen

## Expected behavior

Talkback should focus various part of the screen, like on the main screen. This used to be the case until very recently.

## Actual behavior

Talkback just makes bump sounds and the focus does not move anywhere.

### Device

Xiaomi Redmi Note 12 pro 5G, Motorola Moto g 5G plus

### Android version

14, 10 

### Signal version

8.27.1, 8.27.1

**Comment by ssawczyn** [comment]
I am also a blind TalkBack user and can verify that I am experiencing this same issue.  I believe this issue may have started with the version referenced in the original post.

**Comment by mtang-signal** [comment]
Hi, could you let me know if this issue is resolved with 8.28 (currently being rolled out to production)?
